lib/imapoptions

```diff
@@ -3178,6 +3178,11 @@ product version in the capabilities
    of UID, "cyrus" is UIDVALIDITY.UID.  Dovecot is 8 digits of leading
    hex (lower case) each UID UIDVALIDITY. Courier is UIDVALIDITY-UID. */
 
+{ "twom_checksum_engine", "xxh64", STRING, "UNRELEASED" }
+/* The default checksum engine for new twom databases, string values
+   are currently "null", "crc32", "xxh32", "xxh64".  The default if
+   not set is xxh64. */
```

Comment:
“if not set” is redundant.  As the default value is anyway shown in the rendered documentation, there is no need to spell it explicitly (cf. https://github.com/cyrusimap/cyrus-imapd/pull/4818).

Reply:
Actually this whole thing isn't being used, I'll just strip it.